### PR TITLE
Add the White Gilgamesh

### PR DESCRIPTION
**Pull request by Veritius**
it's a drink. replaces bilk.
also changes the localisation of the Martinez a bit, because grammar

probably will fail tests, i can't check it myself lol

## About the PR

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added the White Gilgamesh.
- remove: Removed bilk.
- fix: Fixed the grammar of The Martinez.

### CODE REVIEW

**Comment by mirrorcult** [comment]
do we not already have a recipe for bilk which is the same thing

**Comment by mirrorcult** [comment]
that said. if we do, replace bilk with this because this is funnier

**Comment by Veritius** [comment]
o shit we do, didn't know that
i'll remove it

File: Resources/Prototypes/Recipes/Reactions/drinks.yml [hunk 1]
```diff
@@ -878,3 +868,13 @@
...
+
+- type: reaction
+  id: WhiteGilgamesh
+  reactants:
+    Beer:
+      amount: 2
+    Milk:
+      amount: 1
+  products:
+    WhiteGilgamesh: 2
```
Comment (crazybrain23) [inline Resources/Prototypes/Recipes/Reactions/drinks.yml:880]: I feel like 3 reagents becoming 2 violates some kind of convention on drinks.

File: Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml [hunk 2]
```diff
@@ -393,3 +393,17 @@
   metamorphicSprite:
     sprite: Objects/Consumable/Drinks/the_martinez.rsi
     state: icon
+
+- type: reagent
+  id: WhiteGilgamesh
```
Comment (crazybrain23) [inline Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml:398]: Should this not be in the alcohol.ftl instead, because it's made from beer?

**Review by crazybrain23 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Reactions/drinks.yml (hunk 1 above)
Comment (Veritius) [reply Resources/Prototypes/Recipes/Reactions/drinks.yml:880]: fuck i forgot that bit, i'll fix it

File: Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml (hunk 2 above)
Comment (Veritius) [reply Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml:398]: it's not alcoholic, but i should probably make it that

**Comment by Veritius** [comment]
the test fail looks completely unrelated to this pr

**Pull request merged**